src/instruction/utils.ts

```diff
@@ -1,39 +1,83 @@
-import { BN } from "@project-serum/anchor";
+import { web3, BN } from "@project-serum/anchor";
 
-export function convertNumbersToBNs(data: Object | any[], keysToUpdate?: string[]) {
+export function convertNumbersToBNs(
+  data: Object | any[],
+  keysToUpdate?: string[]
+) {
+  return convertTypeToType(
+    data,
+    keysToUpdate || [],
+    "number",
+    (value) => new BN(value)
+  );
+}
+
+export function convertStringsToPublicKeys(
+  data: Object | any[],
+  keysToUpdate?: string[]
```

Comment:
Same as this comment https://github.com/raindrop-studios/sol-kit/pull/1/files#r922428361